README.md

```diff
@@ -52,6 +52,12 @@ Using a [standalone script](https://github.com/pnpm/self-installer#readme):
 curl -L https://unpkg.com/@pnpm/self-installer | node
 ```
 
+On Windows (PowerShell):
+
+```powershell
+(Invoke-WebRequest 'https://unpkg.com/@pnpm/self-installer').Content | node
```

Comment:
Please duplicate it in this readme as well: https://github.com/pnpm/pnpm/blob/master/packages/pnpm/README.md